docs/proposals/compute-resource-metrics-api.md

```diff
@@ -61,13 +77,53 @@ kube-ui-v1-nd7in           0.07 cores  130 MB
 A second user will be the scheduler. To assign pods to nodes efficiently, the
 scheduler needs to know the current free resources on each node.
 
-## Proposed endpoints
+The Kubelet API will be used by heapster to provide metrics at the
+cluster-level. The Kubelet API will also be useful for debugging individual
+nodes, and stand-alone kubelets.
 
-	/api/v1/namespaces/myns/podMetrics/mypod
-	/api/v1/nodeMetrics/myNode
+## Proposed endpoints
 
-The derived metrics include the mean, max and a few percentiles of the list of
-values.
+The metrics API will be its own [API group](api-group.md), and is shared by the
+kubelet and cluster API.  All endpoints are GET endpoints, rooted at
+`/apis/metrics/v1alpha1/`
+
+- `/` - discovery endpoint; type resource list
+- `/nodes` - host metrics; type `[]metrics.Node`
+  - `/nodes/localhost` - When served by kubelet, the only node provided is
+    `localhost`; type metrics.Node
+  - `/nodes/{node}` - metrics for a specific node
+- `/derivedNodes` - host metrics; type `[]metrics.DerivedNode`
```

Comment:
Can we document here the difference between /nodes and /derivedNodes? Same for /pods?

From the document, i cannot see any difference.